### PR TITLE
Fix spelling in comments, strings and documentation

### PR DESCRIPTION
**Pull request by ottok**
Fix multiple minor spelling errors in comments, a few strings and in documentation. There are no functional changes.

Please apply the patch in whole or give feedback so I can update the patch. Do not attempt to apply this patch partially – rather give me a chance to make the patch perfect for you.

Signed-off-by: Otto Kekäläinen otto@seravo.fi

### CODE REVIEW

File: docs/reference/commandline/search.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ This example displays images with a name containing 'busybox':
     ofayau/busybox-libc32            Busybox with 32 bits (and 64 bits) libs         1                    [OK]
     peelsky/zulu-openjdk-busybox                                                     1                    [OK]
     skomma/busybox-data              Docker image suitable for data volume cont...   1                    [OK]
-    elektritter/busybox-teamspeak    Leightweight teamspeak3 container based on...   1                    [OK]
+    elektritter/busybox-teamspeak    Lightweight teamspeak3 container based on...    1                    [OK]
```
Comment (thaJeztah) [inline docs/reference/commandline/search.md]: hehe, looks like that typo is on their side;
https://hub.docker.com/r/elektritter/busybox-teamspeak/

**Comment by thaJeztah** [comment]
LGTM, very nice!

**Comment by ottok** [comment]
The failing documentation test stacktrace is not very helpful: https://jenkins.dockerproject.org/job/docs-docker-pr/10266/testReport/junit/(root)/hugo/Tests/
I suspect the test is broken, not my commit.

**Comment by thaJeztah** [comment]
@ottok yeah, it's not related to your changes, it's a bug in hugo; https://github.com/spf13/hugo/issues/1754. I'll restart

**Comment by thaJeztah** [comment]
retest this please

**Comment by vdemeester** [comment]
LGTM 🐯

**Comment by tiborvass** [comment]
@ottok I lived 5 months in Tampere! :) #TUT

**Pull request merged**